assignment-2/7s.md

```diff
@@ -1,2 +1,91 @@
+**Style**: How do the leaders interface with subordinates and others in the organization, and how do members interact with each other?. Aiming to identify typical behavior patterns of key groups.
+**Shared values**: Are the underlying principles or ideas that provide meaning and purpose in an organization.
+
+* *Limitations* .-
+Is there a recepy to follow for example when something wronge is identify?
+
+## 2. Through the eyes of 7S model.
+
+* **2.1 Software industry:**
+
+### Spotify
+
+Spotify is a commercial music streaming, podcast and video service, founded in april 2006 by Daniel Ek, Martin Lorentzon. Spotify was launched in September 2008 and by June 2015 Spotify had more than 75 million active users, including about 20 million paid users. 
+
+##### Strategy
+
+In order to answer the question, What make spotify special?, we could say that they are revolutionizing music industry, making it available with a simple app at your mobil phone or many other electronic devices.
+The strategy of Spotify is based on a freemium business model, with two music streaming tiers: Spotify Free ( a lot advertisements) and Spotify Premium ( paid, remove advertisements, improve audio quality, download available). Spotify pays artists based on their 'market share' (the number of streams for their songs as a proportion of total songs streamed on the service). They distribute approximately 70% to rights-holders.
+
+##### Structure
+
+Spotify headquarters resides in London and its handles, research and development departments are in Stockholm.
+Like many other matrix management organizations, Spotify subdivides the work among people with similar skills, and they are pooled for work assignments, resulting in more than one manager with their own responsibilities and competencies. All this managers are from that point, reporting hierarchical button up to the CTO and finally to the CEO.
+
+##### Systems
+
+How things get done? Spotify has a culture based on agile principles, all engineerings happens in squads, they try to keep those squads loosely coupled, tightly aligned. They like cross-pollination in order to spread knowledge and tools. Their code is shared on an internal open source model. Squads make small and frequent releases as a routine, which is enable due decoupling their services. Their self-service model minimizes the need for handoffs. Their uses release trains and feature toggles to get things into production early and often.
+Their culture is based also on the people, there for their focus on motivation, community and trust rather than structure and control.
+Product development is based on Lean. They release their new features to a small group of customers limiting the 'blast radius' of a possible fail and it allows them to conduce a lot of small experiments. Of course it produces a lot of waste but this can be seen from a Lean perspective.
```

Comment:
Is it possible to have a further explanation about the LEAN base? I do not see a very clear relationship.